app/views/shared/_footer_lite.html.erb

```diff
@@ -61,7 +61,8 @@
                   <li class='border-bottom border-navy'>
                     <%= link_to t("i18n.locale.#{locale}"),
                       sanitized_requested_url.merge(locale: locale),
-                      class: 'block pl-24p padding-y-2 text-decoration-none white' %>
+                      class: 'block pl-24p padding-y-2 text-decoration-none white language',
+                      lang: locale %>
```

Comment:
Instead of removing this via JS when it's the current language, what if we did that all server-side?

```suggestion
                      lang: locale == I18n.locale ? nil : locale %>
```

Reply:
@zachmargolis Wouldn't this render as `nil` because I18n.locale is always true?

Reply:
only when `locale == I18n.locale`? and rails will hide `nil` values from being rendered so I think it's OK

Reply:
Alternatively, is there any harm to just rendering the `lang` even if it's the same as the current language?

Reply:
@aduth I don't _think_ there is anything wrong with that, but based on what I read, it seems unnecessary.

Reply:
> @aduth I don't _think_ there is anything wrong with that, but based on what I read, it seems unnecessary to do that.

Sounds good. Would have been more appealing when the implementation was using JavaScript to remove the current language, but since the revised implementations looks pretty simple enough as it is, seems fine either way 👍